### DISCUSSION TITLE
Wrong websocket url

### DISCUSSION THREAD

**Discussion by EvgenyLyutikov** [embl-bio-it/python-mattermost-autodriver#10] [Q&A] [answered]
Hello!
as described in [api reference](https://api.mattermost.com/#tag/WebSocket) url for websocket connection should be `/api/v4/websocket` but when we conctruct websocket client URL build from empty basepath and it wrong `wss://host:443/websocket`

**Comment by unode** [comment, marked as answer]
Can you provide additional details, which version of mattermostautodriver? A related issue was resolved in 1.3.0 and shouldn't be present in 2.0.0 either.

**Comment by EvgenyLyutikov** [comment]
mattermostautodriver==1.2.4
Thanks, I'll try the latest version

**Reply by EvgenyLyutikov** [reply]
Worked correct on latest version
